### DISCUSSION TITLE
Marking table as "Date Table"

### DISCUSSION THREAD

**Discussion by Declan1984** [TabularEditor/TabularEditor3#393] [🙋♀️TE3 Q&A] [answered]
When I connect to my model in Dax Studio, the "Calendar" table has a clock icon on it and it is described as "Marked as Date Table".  Within Tabular Editor, is this setting of "Marked as Date Table" controlled by the table Properties => Options => Data Category == "Time" or is it designated in some other way? @otykier

**Comment by otykier** [comment, marked as answer]
Hi Michael

In Tabular Editor, we provide the following option when you right-click on a table:

![image](https://user-images.githubusercontent.com/8976200/152772769-9c29d52d-2dc9-4e7c-85dd-9cc2e0fc424d.png)

This sets the data category property of the table to "Time", and requires you to select a DateTime column on the table to be marked as "Key", which is required for a table to appear as a date table in client tools such as DAX Studio and Power BI.
